src/app/components/Drbb/DrbbItem.jsx

```diff
@@ -47,14 +44,28 @@ const DrbbItem = (props) => {
     );
   };
 
+  let downloadLink;
+
+  const selectEdition = () => (editions.find(workEdition => (
```

Comment:
The edition items are sorted by most useable to least useable, so you could take `edition.items[0]`.  
In DRB we use https://github.com/NYPL/sfr-bookfinder-front-end/blob/development/src/app/components/Detail/WorkDetail.jsx#L75

Reply:
There's a possibility that work.editions[0] is always the edition with the most data, (edition.instances[0] is always featured ... @mwbenowitz is this true?)  We're not taking advantage of it in DRB but there's no reason why you shouldn't ...

Reply:
That isn't strictly true, `editions[0]` always has at least one link, but that's all. Hypothetically there are better editions within the work detail. In reality it is functionally true, especially since the quality doesn't vary much between editions.

However, taking `editions[0]` would make the "featured" copy in SCC and DRB the same, which is a consideration I hadn't thought of before

Reply:
Do either of you have strong feelings towards getting rid of/changing this logic?

Reply:
I'd be fine with it, it would bring the SCC integration more in line with how DRB presents these records

Reply:
I think this logic becomes much simpler if you use `editions[0]`, which is nice.